src/sidebar/components/autocomplete-list.js

```diff
@@ -12,6 +30,8 @@ const defaultListFormatter = item => item;
  * used by itself.
  *
  * Modeled after the "ARIA 1.1 Combobox with Listbox Popup"
+ * @template T
+ * @param {AutocompleteListProps<T>} props
  */
 
 export default function AutocompleteList({
```

Comment:
Style nit: I don't think TS cares, but for JSDoc that refers to a specific function/variable etc. there shouldn't be a new line between the JSDoc comment and the thing it refers to.

Reply:
good to know, thanks